docs/getting_started.md

```diff
@@ -0,0 +1,40 @@
+1. Inspect your dataset
+-----------------------
+The Study Template **only** works with BIDS-formatted raw data.
```

Comment:
a link to BIDS specification and maybe a link to MNE-BIDS so they know how to convert their data into BIDS?

Reply:
Done!